templates/documents/general_view.html

```diff
@@ -309,8 +309,43 @@ <h4>{xl t='Contents'|escape:'html'}</h4>
 			 $file->get_mimetype() eq "image/gif" or
 			 $file->get_mimetype() eq "application/pdf" }
 			<iframe frameborder="0" type="{$file->get_mimetype()}" src="{$web_path}as_file=false"></iframe>
-			{elseif $file->get_ccr_type($file->get_id()) ne "CCR" and
-       $file->get_ccr_type($file->get_id()) ne "CCD"}
+            {elseif $file->get_mimetype() eq "application/dicom"}
+            <!-- DWV -->
+            <div id="dwv" src="{$web_path}as_file=false">
+                <!-- Toolbar -->
+                <div class="toolbar"></div>
+                <div class="warn_diagnostic">Not For Diagnostic Use</div>
+                <!-- Layer Container -->
+                <div class="layerContainer">
+                    <canvas class="imageLayer">Only for HTML5 compatible browsers...</canvas>
+                </div><!-- /layerContainer -->
+            </div><!-- /dwv -->
+            <!-- Third party (dwv) -->
+            <script type="text/javascript" src="{$assets_static_relative}/modernizr-3-5-0/dist/modernizr-build.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-9-0-1/i18next.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-9-0-1/i18next.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-xhr-backend-1-4-3/i18nextXHRBackend.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-browser-languagedetector-2-0-0/i18nextBrowserLanguageDetector.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/konva-1-6-8/konva.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/magic-wand-js/js/magic-wand-min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/jszip-3-1-5/dist/jszip.min.js"></script>
+            <!-- Third party (viewer) -->
+            <script type="text/javascript" src="{$assets_static_relative}/jquery-ui-1-12-1/jquery-ui.min.js"></script>
```

Comment:
This jquery-ui-1-12-1 is not needed, since it's already included (this other script is the header):
https://github.com/openemr/openemr/blob/master/templates/documents/general_list.html#L40